### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/graph/Graph.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  private Map<Vertex<T>, ArrayList<Edge<T>>> adjacencyList;
+  
+  public Graph() {
+    this.adjacencyList = new HashMap<>();
+    this.size = 0;
+  }
+  
+  public Graph(Map<Vertex<T>, ArrayList<Edge<T>>> adjacencyList) {
+    this.adjacencyList = adjacencyList;
+    this.size = 0;
```
Comment (evanslaton) [inline code401challenges/src/main/java/graph/Graph.java]: What if the adjacency list passed has a size greater than 0?
